ra_aid/__main__.py

```
@@ -174,26 +209,32 @@ def main():
 
         if expert_missing:
-            console.print(Panel(
-                f"[yellow]Expert tools disabled due to missing configuration:[/yellow]\n" +
-                "\n".join(f"- {m}" for m in expert_missing) +
-                "\nSet the required environment variables or args to enable expert mode.",
-                title="Expert Tools Disabled",
-                style="yellow"
-            ))
+            console.print(
+                Panel(
+                    f"[yellow]Expert tools disabled due to missing configuration:[/yellow]\n"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)